site/docs/sql/00-intro.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Hazelcast Jet SQL
+description: Introduction to Hazelcast Jet SQL features.
+---
+
+Hazelcast Jet allows you to create a Jet processing job using the
+familiar SQL language. It can execute distributed SQL statements over
+Hazelcast IMaps and external data sets.
+
+**Note:** _The service is in beta state. Behavior and API might change
+in future releases. Binary compatibility is not guaranteed between minor
+or patch releases._
+
+## Overview
+
+In the first release, Jet SQL supports the following features:
+
+- SQL Queries over [Apache Kafka topics](05-kafka-connector.md) and
+[files (local and remote)](04-files-connector.md)
+- Joining Kafka or file data with local IMaps (enrichment)
+- Filtering and projection using [SQL
+expressions](https://docs.hazelcast.org/docs/{imdg-version}/manual/html-single/index.html#expressions)
+- Aggregating data from files using predefined
```

Comment:
"Aggregating data from files" - data from Kafka cannot be aggregated?

Reply:
Nope, you can't aggregate a streaming data - you need complete data and they are never complete. For imap it doesn't work either, but in this case just because it isn't implemented yet.